src/poetry/installation/installer.py

```diff
@@ -258,10 +258,9 @@ def _do_install(self, local_repo: Repository) -> int:
             if not self._locker.is_fresh():
                 self._io.write_line(
                     "<warning>"
-                    "Warning: The lock file is not up to date with "
-                    "the latest changes in pyproject.toml. "
-                    "You may be getting outdated dependencies. "
-                    "Run update to update them."
+                    "Error: poetry.lock is not consistent with pyproject.toml. "
```

Comment:
This should also start with `"Warning: "`

Reply:
Updated